utils/runner.go

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import (
+	"fmt"
+)
+
+// Prints the specified app version to STDOUT
+func PrintVersion(appName string, version string, gitref string) {
+	if gitref != "" {
+		fmt.Printf("%v version %v, gitref: %v\n", appName, version, gitref)
+	} else {
+		fmt.Printf("%v version %v\n", appName, version)
+	}
```

Comment:
Can we make this match the `teleport version`/`tctl version`/`tsh version` output format?

Something more like:

```go
// PrintVersion prints the specified app version to STDOUT
func PrintVersion(appName string, version string, gitref string) {
	if gitref != "" {
		fmt.Printf("%v v%v git:%v %v\n", appName, version, gitref, runtime.Version())
	} else {
		fmt.Printf("%v v%v %v\n", appName, version, runtime.Version())
	}
}
```